chia/cmds/farm_funcs.py

```diff
@@ -181,69 +181,101 @@ async def challenges(farmer_rpc_port: int, limit: int) -> None:
         )
 
 
-async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int, farmer_rpc_port: int) -> None:
+class Summary:
+    def __init__(self):
+        self.status = None
+        self.total_chia_farmed = None
+        self.user_transaction_fees = None
+        self.block_rewards = None
+        self.last_height_farmed = None
+        self.total_plot_size = None
+        self.plot_count = None
+        self.estimated_network_space = None
+        self.expected_time_to_win = None
+
+    def __str__(self):
+        def format(name: str, value: str) -> str:
+            output = f"{name}: "
+            if value is None:
+                output += "Unknown"
+            else:
+                output += f"{value}"
+            output += "\n"
+            return output
+
+        output = format("Farming status", self.status)
+        output += format("Total chia farmed", self.total_chia_farmed)
+        output += format("User transaction fees", self.user_transaction_fees)
+        output += format("Block rewards", self.block_rewards)
+        output += format("Last height farmed", self.last_height_farmed)
+        output += format("Plot count", self.plot_count)
+
+        output += "Total size of plots: "
+        if self.total_plot_size is None:
+            output += "Unknown"
+        else:
+            plots_space_human_readable = self.total_plot_size / 1024 ** 3
+            unit = "GiB"
+            if plots_space_human_readable >= 1024 ** 2:
+                plots_space_human_readable = plots_space_human_readable / (1024 ** 2)
+                unit = "PiB"
+            elif plots_space_human_readable >= 1024:
```

Comment:
You could also get rid of the elif in favour of a second if and thus drop the two **2's